softwareconfiguration/autofe-reldata/src/main/java/autofe/db/search/DatasetEvaluationFailedException.java

```diff
@@ -0,0 +1,14 @@
+package autofe.db.search;
+
+public class DatasetEvaluationFailedException extends RuntimeException {
+

```

Comment:
I think that this kind of exceptions should be checked (not inherit from RuntimeException)

Reply:
Resolved